bin/socorro_env.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# Launches bash in a socorro-ized environment to use on the servers. This
+# includes the Socorro Python virtualenv as well as pulling in consul
+# configuration.
+#
+# This doesn't work in docker and isn't needed.
+#
+# Usage:
+#
+#     /data/socorro/bin/socorro_env.sh
+
+set -e
+
+echo "Activate Python virtualenv..."
+# Active virtualenv in this environment
+. /data/socorro/socorro-virtualenv/bin/activate
+
+echo "Add all the stuff from envconsul..."
+export "$(envconsul -prefix socorro/common -prefix socorro/processor -prefix socorro/webapp-django -prefix socorro/crontabber env)"
```

Comment:
Neato! 

The description you mentioned in the PR description isn't really fair though. Those are different things. If one of the commands you start with `envconsul` is long-running (or daemon) it'll be restarted if you change any config in any of the connected nodes. This new approach is different in that it won't update once set. So it's only really useful for one-off script executions. 

Perhaps it might be worth, as an optional, nit to remind of that. Either in an `echo` or as a doc string here in this script.